front/src/App.tsx

```diff
@@ -54,10 +62,42 @@ function Protected({ component, redirectPath, requiredAbp, minBeta }: IProtected
     />
   );
 }
-Protected.defaultProps = {
-  requiredAbp: [],
-  minBeta: 0,
-};
+
+/**
+ * Handle Dropapp (Boxtribute V1 app) redirects whose paths don't start with `/bases/:baseId`.
+ *
+ * Fetch first available base id from user JWT token from Auth0 to prepend `/bases/:baseId` with that id, if available.
+ */
+function DropappRedirect({ path }: DropappRedirectProps) {
+  const { user } = useAuth0();
+  let pathToRedirect = "/error";
```

Comment:
I do not think that we have this path

Reply:
The reason for this is to hit the `<NotFoundView />` Route. I'll leave a comment to make this more explicit.